utils/hv-tools/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hv-tools
+PKG_VERSION:=1.0.0
```

Comment:
This version is tagged, but it leads to https://github.com/NikulovE/hv-tools/commit/126d8bf6434c76e1db4d180bd67de89141f5f618 , but this commit is not used below.

You need to decide, which version do you want to use.

Reply:
Yes, I switched from 2024.08.01 to 1.0.0 and will not change it again.